source/Ship.cpp

```diff
@@ -1149,6 +1151,21 @@ int64_t Ship::ChassisCost() const
 
 
 
+// Get the full cost of this ship on the planet, depending on the outfit prices.
+int64_t Ship::LocalCost(const Planet *planet, const ConditionsStore &conditions) const
+{
+	int64_t localCost = ChassisCost();
+	const map<CustomSale::SellType, CustomSale> sales;
+	if(planet)
+		GameData::GetCustomSales(*planet, conditions);
```

Comment:
This doesn't look like it does anything?
Is this trying to force a cache refresh or something?